src/Command/PelagosMigrateDatasetFilesCommand.php

```diff
@@ -177,7 +175,7 @@ protected function setFile(
 
             $file->setDescription('Original Dataset File');
 
-            $file->setPhysicalFilePath($fileName);
+            $file->setPhysicalFilePath("$udi/$udi.dat");
```

Comment:
This is where the change needed to happen, because the path is now relative.
The `$fileName` above is also used for checking if the name exists.